doc/releases/index.rst

```diff
@@ -1,14 +1,66 @@
 .. _zephyr_release_notes:
 
-Release Notes
-#############
+Releases
+########
 
 Zephyr project is provided as source code and build scripts for different
 target architectures and configurations, and not as a binary image. Updated
 versions of the Zephyr project are released approximately every three-months.
 
 All Zephyr project source code is maintained in a `GitHub repository`_.
 
+Release Life Cycle and Maintenance
+**********************************
+
+Periodic Releases
+=================
+
+The Zephyr project provides periodic releases every 4 months leading to the
+long term support releases approx. every 2 years. Priodic and non-LTS releases
+are maintained with updates, bug fixes and security related updates for at
+least two cycles, meaning that the project supports the most recent two
+releases in addition to the most recent LTS.
+
+Long Term Support and Maintenance
+=================================
+
+A Zephyr :ref:`Long Term Support (LTS) <release_process_lts>` release is
+published every 2 years and is branched and maintained independently from the
+main tree for at least 2.5 years after it was released.
+
+Support and maintenance for an LTS release stops at least half a year
+after the following LTS release is published.
+
+Security Fixes
+==============
+
+Each security issue fixed within zephyr shall be backported to the
+following releases:
+
+- The current Long Term Stable (LTS) release.
```

Comment:
According to the above:

> Support and maintenance for an LTS release stops at least half a year after the following LTS release is published.

So, I guess that security fixes happening during the first 6 months after an LTS release would also make it into the previous LTS? Or am I understanding that wrong?

Thanks!

Reply:
yes, if they apply to the previous release.

Reply:
In that case I would say something like:

> Each security issue fixed within zephyr shall be backported to the following releases:
> - The current Long Term Stable (LTS) release.
> - The previous Long Term Stable (LTS) release if within 6 months after the current LTS release date.
> - The most recent two releases.

Would that work? Thanks.

Reply:
I addressed that by tweaking the first bullet.

Reply:
That's easier indeed. Thanks!